### PR TITLE
Calls to sntp adjusted to syntax from ntp 4.2.8 (FATE#320394)

### PR DESCRIPTION
**Pull request by teclator**
It is a backport from #28 and #46.

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.17.14
+2.17.14.1
```
Comment (imobach) [inline VERSION]: Same version than Code-11-SP1? Check #47.

Comment (imobach) [inline VERSION]: Ok, so no difference between both branches.

**Comment by imobach** [comment]
LGTM

File: VERSION (hunk 1 above)
Comment (teclator) [inline VERSION]: No, it is the same code for both and diverge in SP3.

**Pull request merged**